### DISCUSSION TITLE
Fullstack .NET : MudDataGrid sem paginacao

### DISCUSSION THREAD

**Discussion by LuizFredericoJr** [balta-io/forum#641] [Tira Dúvidas] [answered]
Balta, boa tarde.
Mesmo mudando o tamanho da pagina para 5, a paginacao nao funciona...
No site do Mud, vi que precisa ter isso depois do columns/>
```

"<PagerContent>"
"    <MudDataGridPager T="Element" />"
"</PagerContent>"

```
Eh isso mesmo? vc poderia ajustar isso nos lists das paginas?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @LuizFredericoJr, como vai?

Acredito que tenha alguma configuração que precisa ser feita:
https://mudblazor.com/components/pagination

Vou tentar cobrir isto nas próximas aulas!
